### ISSUE TITLE
Standardizing API responses

### ISSUE THREAD

**Issue by lwojcik** [jointwt/twtxt#141]
From [API docs](http://dev.twtxt.net/doc/api.html):

> Serialization is JSON. Every request requires a JSON payload even if it is empty. Most respones will send a JSON payload as their response but not all. (some may respond with an empty body and just a HTTP status).

Is there a reason we opted for empty responses instead of, e.g.,  empty JSON objects or something generic like `{"status":"200","message":"ok"}`?

Coding in JavaScript and consuming API responses with Fetch API I'm a bit unhappy about this. It requires me to pass response object through additional endpoint-specific safeguards even if response was 200 OK, which adds complexity. I already have code in place to do so; I could also use external library like axios, but I find it a bit overkill for handling such a minor problem.

It's not a burning issue for now but I'd welcome standardizing API behaviors in future API iterations.

**Comment by prologic** [comment]
Yeah okay I'll fix this. Should be easy to do! It _will_ however break the current API so cc @dooven @aaadonai You guys will want to update your apps rather quickly, or in fact I should maybe wait for you two before making this change? The alternative is I figure out how to make an `/api/v2` but that's kind of overkill IHMO at this stage.

**Comment by aaadonai** [comment]
I don't think we need to go to /v2/ just yet. The apps are only in invite private beta. It is ok to break.

**Comment by lwojcik** [comment]
Now that I publicly released a [JavaScript library](https://github.com/jointwt/twt.js), which I treat as production-ready despite 0.x.x version number and I plan to build more stuff on top of it, I want to state I'm okay with breaking changes in the API but I need to be able to consult a single source of truth whenever it happens (at least one document, hopefully [API docs](http://dev.twtxt.net/doc/api.html) should reflect current state 100% of the time). That way I'll be able to plan accordingly and release updates as quick as I can. I can also help updating API docs whenever it happens, just keep me in the loop should breaking changes happen.

**Comment by prologic** [comment]
Bah this issue was closed in error by the stupid bot 😂
